vm-setup/roles/v1aX_integration_test/tasks/move.yml

```diff
@@ -1,9 +1,24 @@
 ---
+  - name: Remove ironic container from source cluster
+    vars:
+      ansible_python_interpreter: /usr/bin/python3
+    docker_container:
+      name: "{{ item }}"
+      state: absent
+    with_items:
+       - ironic
+       - ironic-inspector
+       - dnsmasq
+       - httpd
+       - mariadb
+       - ironic-endpoint-keepalived
```

Comment:
this is specific to Kind, if someone would attempt to run this with minikube would it fail ? If not failing, at least it would not undeploy ironic on Minikube. Until now everything was pretty much agnostic of the source cluster, but with this, you either need to tackle both or make it really clear that this is not supposed to be run with Minikube. I would much prefer the first option tbh.

Reply:
Please see the above comment.

Reply:
I have added additional code and check for minikube, pleas review